src/olympia/accounts/urls.py

```diff
@@ -1,15 +1,17 @@
-from django.conf.urls import include, url
+from django.conf.urls import include
+from django.urls import re_path
 
 from rest_framework.routers import SimpleRouter
 from rest_framework_nested.routers import NestedSimpleRouter
 
-from olympia.bandwagon.views import CollectionAddonViewSet, CollectionViewSet
+from olympia.bandwagon.views import CollectionAddonViewSet, \
+    CollectionViewSet
```

Comment:
random extra change here.

Reply:
actually, in quite a few places - it seems like you've over-compensated on line length?

Reply:
I was getting code style test error as in some of the lines , the length was exceeding the 79 characters limit. So I reformatted
the files with limit of 70 characters.

Reply:
yes, 79 characters is what we want

Reply:
Changes are done with maximum limit of 79 characters.